Fomo/TripEvent.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  TripEvent.swift
+//  Fomo
+//
+//  Created by Christian Deonier on 2/29/16.
+//  Copyright © 2016 TeamAwesome. All rights reserved.
+//
+
+import UIKit
+
+class TripEvent: NSObject {
+    
+    enum Vote: Int {
+        case Dislike = 1, Neutral, Like
+    }
+
+    var id: Int?
+    var attraction: Attraction?
+    var startTime: NSDate?
+    var endTime: NSDate?
```

Comment:
I think this could be a good idea-- maybe if we decide how many slots each day has beforehand?